### ISSUE TITLE
"Create proccess rate" property should not change a PK-Sim module to an extension module

### ISSUE THREAD

**Issue by PavelBal** [Open-Systems-Pharmacology/MoBi#1443]
E.g. in a passive transport "Create process rate".

**Comment by rwmcintosh** [comment]
and in other builders

**Comment by benjaperez1983** [comment]
![image](https://github.com/user-attachments/assets/aebc0f81-e21e-42a6-adb9-fb962aded96a)

**Comment by benjaperez1983** [comment]
![image](https://github.com/user-attachments/assets/33a52183-4163-4f8e-b6d8-7d1b123bcf51)
Shall this uncheck action make this red?

**Comment by PavelBal** [comment]
No, I do not think that un/selecting the "Create process rate parameter" should mark the simulation as out of synch. The reason is that we do not support "update from building block" any more, and when comparing the simulation with the template building block, no differences are shown.

Instead, I propose to always show the "Update from BBs" action for a simulation.

**Comment by Yuri05** [comment]
>  I do not think that un/selecting the "Create process rate parameter" should mark the simulation as out of synch. 

I think it should. Especially when unselecting, simulation might be not even creatable after that (e.g. because the ProcessRate parameter is used elsewhere).

**Comment by msevestre** [comment]
I agree with @Yuri05 Whatever changes the model structure should part the simulation has changed. This is how it's working everywhere else no? why should the behaviour of this flag be different?

**Comment by PavelBal** [comment]
Okay, agree, but in this case, this should be reflected in the comparison somehow.

**Comment by Yuri05** [comment]
I thought we agreed that changing of "Create process rate" SHOULD change a PK-Sim module to an extension module?
But currently it does not.

**Comment by Yuri05** [comment]
If I say **No** in the conversion prompt: the value of "Create process rate" is still changed (even though the module remains as PK-Sim module). It's not just a refresh issue: after closing/reopening the project the value ist still changed.
There is also no new command added in the history, so very confusing why the changed value is kept...

![Image](https://github.com/user-attachments/assets/3d1a0563-0f5b-4a7c-8ee7-04bb2c93b791)

**Comment by PavelBal** [comment]
I still really do not like this. With this implementation, there is no way to debug the model (see the rates of processes) without converting the module to an extension module.

**Comment by Yuri05** [comment]
> I still really do not like this. With this implementation, there is no way to debug the model (see the rates of processes) without converting the module to an extension module.

Switching process rates on/off will in some cases change the model outputs (e.g. base PK-Sim models use sum of process rates for calculation of fabs etc.).

For debugging, you could create a clone of PK-Sim module

**Comment by PavelBal** [comment]
> For debugging, you could create a clone of PK-Sim module

Yes okay, or use the "create all process rate parameters" checkbox during simulation creation.
